modules/telco-ran-414-whats-new-ref-design.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-design-spec.adoc
+
+:_content-type: CONCEPT
+[id="telco-ran-whats-new-ref-design_{context}""]
+= What's new in {product-title} {product-version} for telco RAN
+
+The following new features included in {product-title} {product-version} are leveraged by the telco RAN DU reference design.
+
+.New features for telco RAN in {product-title} {product-version}
+[cols="1,3", options="header"]
+|====
+|Feature
+|Description
+
+//CNF-7078
+|Intel e810 NIC as PTP Grandmaster clock
+a|You can use the Intel E810-XXV-4T NIC as a GNSS-sourced grandmaster clock.
+The NIC is automatically configured by the PTP Operator with the E810 hardware plugin.
+This feature is scheduled for an upcoming 4.14 z-stream release.
```

Comment:
Not sure it would in this instance. I use `{product-version}` when I know the content is going to live beyond the current release. This is a "point in time" doc so IMO perfectly acceptable to use "4.14" here.